internal/web_server/web/views/components/dashboard_components/user_admin/username_tables.templ

```diff
@@ -0,0 +1,12 @@
+package user_admin
+
+import "github.com/Molnes/Nyhetsjeger/internal/models/users/usernames"
+
+
+// Tables of the two components an username is created from: Nouns and adjectives
+templ UsernameTables(uai *usernames.UsernameAdminInfo) {
+	<div id="tables-wrapper" class="flex gap-3">
+		@WordTable(uai.Adjectives, "Adjektiv", uai.APage, uai.AWordCount, "adjective-table", "apage")
+		@WordTable(uai.Nouns, "Substantiv", uai.NPage, uai.NWordCount, "noun-table", "npage")
```

Comment:
Again, no way with NWordCount,
Also with the variable name maybe written out

Reply:
changed names